packages/site-components-next/package.json

```diff
@@ -36,10 +36,15 @@
     "@jpmorganchase/mosaic-types": "^0.1.0-beta.62",
     "clsx": "^2.0.0",
     "deepmerge": "^4.2.2",
-    "rehype-slug": "^5.0.1",
-    "remark-gfm": "^3.0.1",
+    "rehype-pretty-code": "0.13.0",
+    "remark-rehype": "11.1.0",
+    "rehype-slug": "^6.0.0",
+    "rehype-stringify": "10.0.0",
+    "remark-gfm": "^4.0.0",
     "server-only": "^0.0.1",
-    "next-mdx-remote": "^4.4.1"
+    "shiki": "^1.1.1",
+    "@daviereid/next-mdx-remote": "^4.4.2",
```

Comment:
what was this change?

Reply:
update a few dependencies so that they all use `unified` version 11 which is needed by `shiki`.

PR has been raised against the `next-mdx-remote` repo --> https://github.com/hashicorp/next-mdx-remote/pull/433